build-logic/buildquality/src/main/kotlin/gradlebuild/incubation/tasks/IncubatingApiAggregateReportTask.kt

```diff
@@ -26,7 +26,7 @@ import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskAction
-import org.gradle.kotlin.dsl.submit
+import org.gradle.kotlin.dsl.*
```

Comment:
:x: Let's not use `*` imports.

Reply:
* is used, to workaround https://youtrack.jetbrains.com/issue/KTIJ-24390/, another option is to suppress unused import warnings for a file. (or not dogfood assignment) here.

Reply:
added a comment